load_testing/common_flows/flow_helper.py

```diff
@@ -222,3 +273,44 @@ def load_fixture(filename, path="./load_testing"):
         raise RuntimeError(f"Could not find fixture {fullpath}")
 
     return fixture
+
+
+def export_cookies(domain, cookies, savelist=None):
+    """
+    Export cookies used for remembered device/other non-session use
+    as list of Cookie objects.  Only looks in jar matching host name.
+
+    Args:
+        domain (str) - Domain to select cookies from
+        cookies (requests.cookies.RequestsCookieJar) - Cookie jar object
+        savelist (list(str)) - (Optional) List of cookies to export
+
+    Returns:
+        list(Cookie) - restorable using set_device_cookies() function
+    """
+    if savelist is None:
+        savelist = DEFAULT_COOKIE_SAVELIST
+
+    # Pulling directly from internal data structure as there is
+    # no get_cookies method.
+    dcookies = cookies._cookies.get(domain, {}).get('/', None)
+
+    if dcookies is None:
+        return []
+
+    return [c for c in [dcookies.get(si) for si in savelist] if c is not None]
```

Comment:
I sang this line to the tune of "C is for 🍪"

Reply:
Good enough for me.